verify/verify.py

```diff
@@ -337,17 +335,14 @@ async def verify_status(self, ctx: commands.Context):
         embed = discord.Embed(colour=(await ctx.embed_colour()))
         embed.add_field(name="Verified", value=f"{count} users")
 
-        if role_id:
-            role = ctx.guild.get_role(role_id)
+        if role_id and (role := ctx.guild.get_role(role_id)):
```

Comment:
This shouldn't silently fail - Add a message to all the fields that you've made conditional here instead of omitting them when the condition is false.